app-cpp/app355.cpp

```diff
@@ -393,6 +360,45 @@ void SatOrbit::OnSortOrbitalVectorMulti(orbit_iterator& inBegin, orbit_iterator&
     return inLHS.GetTLE().GetMeanMotion() < inRHS.GetTLE().GetMeanMotion();
 }
 
+void SatOrbit::CalculateOrbitalDataMulti(const std::vector<sat355::TLE>& inTLEVector, OrbitalDataVector& ioDataVector)
+{
+    // create a list of threads the size of the number of threads specified
+        std::vector<std::thread> threadVector{};
+        threadVector.reserve(mNumThreads);
+        const std::size_t tleVectorSize = inTLEVector.size();
+
+        // calculate the number of TLEs per thread
+        const std::size_t tlePerThread = tleVectorSize / mNumThreads;
+        auto tleBegin = inTLEVector.begin();
+        auto tleEnd = inTLEVector.end();
```

Comment:
wow. really hard for me to find something to demand a change to in this code...
so, i'm gonna go "super-nit-picky" and suggest this change:
`tleBegin` and `tleEnd` just sorta hang out there duplicating values held by `inTLEVector`
- Remove both `tleBegin` and `tleEnd` from here